src/test/scala/com/microsoft/hyperspace/index/RefreshIndexTests.scala

```diff
@@ -248,4 +329,15 @@ class RefreshIndexTests extends QueryTest with HyperspaceSuite {
 
     fileToDelete
   }
+
+  private def logManager(indexName: String): IndexLogManager = {
+    val indexPath = PathUtils.makeAbsolute(s"$systemPath/$indexName")
+    IndexLogManagerFactoryImpl.create(indexPath)
+  }
+
+  private def fileList(path: String): Seq[String] = {
```

Comment:
`listFiles`? I think that's what this `def` does.

Reply:
thanks, @imback82 , fixed